VMAccess/vmaccess.py

```diff
@@ -79,6 +82,14 @@ def enable():
         if remove_user:
             _remove_user_account(remove_user, hutil)
         _set_user_account_pub_key(protect_settings, hutil)
+	if check_disk:
+            _fsck_check()
+            hutil.log("Successfully checked disk")
+            hutil.exit_if_enabled()
+        if repair_disk:
+            _fsck_repair()
+            hutil.log("Repaired and remounted disk")
+            hutil.exit_if_enabled()
```

Comment:
@Snesha 
Agree with @paulmey .
If you want to exit at line 88 or line 92, you should call hutil.do_exit with right status.